lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -230,6 +236,13 @@ data ByronWalletStyle
     | Icarus
     | Trezor
     | Ledger
+    deriving (Show, Generic, Eq, Bounded, Enum)
+
+instance FromText ByronWalletStyle where
+    fromText = fromTextToBoundedEnum KebabLowerCase
+
+instance ToText ByronWalletStyle where
+    toText = toTextFromBoundedEnum KebabLowerCase
```

Comment:
Side-note: we use `SnakeLowerCase` everywhere else. It has no influence here because all words are single words but it'd be better to be consistent with casing on other parts..

Reply:
It seemed more appropriate for the CLI… which I'll admit might be iffy when the instance itself lives with the Api-code…

Changing to `SnakeLowerCase` since it doesn't matter.